src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -0,0 +1,104 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Elasticity {
+/*!
+ * \brief Constitutive relations for elastic materials
+ */
+namespace ConstitutiveRelations {
+
+/*!
+ * \brief An isotropic and homogeneous material
+ *
+ * \details For an isotropic and homogeneous material the linear constitutive
+ * relation \f$T^{ij}=\chi^{ijkl}S_{kl}\f$ reduces to
+ * \f[
+ * \chi^{ijkl} = \lambda \delta^{ij}\delta^{kl} + \mu
+ * \left(\delta^{ik}\delta^{jl} + \delta^{il}\delta^{jk}\right) \\
```

Comment:
I agree with this equation (it has all the right symmetries, and I use it in my thermal noise code), but I can't seem to find a good reference for it. Do you have one you can cite here?